src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,17 @@
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+
+const booksReducers = (state = [], action) => {
+  const reducerMap = {
+    ADD: () => [...state, action.book],
+    REMOVE: () => state.filter((book) => book.id !== action.id),
+  };
+
+  const reducerFunction = reducerMap[action.type];
+  return reducerFunction ? reducerFunction() : state;
+};
+
+const addBook = (book) => ({ type: ADD, book });
+const removeBook = (id) => ({ type: REMOVE, id });
+
+export { booksReducers, addBook, removeBook };
```

Comment:
- [ ] Kindly use `createSlice` from `@reduxjs/toolkit` to create the actions and the reducers in the `booksSlice`. You can read this [article](https://medium.com/geekculture/understanding-createslice-in-redux-toolkit-reactjs-eca8d20f45d7) to understand how to use `createSlice` to create the actions and reducers. 👍 😎